extension/latextext.py

```diff
@@ -587,8 +586,18 @@ def render(self, latex_code, preamble_file=None, package_list="", fontsize=10, s
         # Convert PDF to SVG
         if PLATFORM == WINDOWS:
             PDF2SVG_PATH = os.path.join(os.path.realpath(EXT_PATH), 'pdf2svg')
+        if PLATFORM == MAC:
```

Comment:
```suggestion
        elif PLATFORM == MAC:
```
unless you hate windows users (which is unfair, they already go through enough as it is :P)